### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by kspieks**
This PR creates a starting point for the T3 documentation. Let me know how you want to handle the version number (I referred to this as v1.0.0). Looking forward to your feedback!

### CODE REVIEW

File: docs/source/contribute.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.. _contribute:
+
+Contribute
+==========
+
+We use T3 to facilitate our research and have made it available
+as a benefit to the community in the hopes that others may find it useful as well.
+
+We welcome contributions from the community. Please see the `contributor guidelines`__
+on T3's GitHub page, as well as the Code of Conduct.
```
Comment (alongd) [inline docs/source/contribute.rst:10]: could you link to the code of conduct file?

File: docs/source/index.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
+T3 - RMG-ARC Tandem Tool v\ |release|
```
Comment (alongd) [inline docs/source/index.rst:1]: without "The", it's not T3 ('The RMG-ARC Tandem Tool")

File: docs/source/release.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.. _release:
+
+Release notes
+=============
+
+T3 1.0.0
```
Comment (alongd) [inline docs/source/release.rst:6]: our current version should be 0.1.0

File: docs/source/release.rst [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+^^^^^^^^^
+
+This is the first stable version of T3. See the API.
```
Comment (alongd) [inline docs/source/release.rst:9]: 0.1.0 is not "stable"

File: docs/source/cite.rst [hunk 5]
```diff
@@ -0,0 +1,10 @@
+.. _cite:
+
+How to cite T3
+===============
+
+Text form::
+
+    version 1.0.0, https://github.com/ReactionMechanismGenerator/T3
```
Comment (alongd) [inline docs/source/cite.rst:8]: should add authors and the repository name

File: docs/source/links.txt [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+.. _GitHub: https://github.com/ReactionMechanismGenerator/T3
+.. _greengp: http://greengroup.mit.edu/
+.. _jupyter: https://jupyter.org/
+.. _DFTGridStudy: https://doi.org/10.26434/chemrxiv.8864204.v5
+.. _mergeConflict: https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts
+.. _RMG-Py: https://github.com/ReactionMechanismGenerator/RMG-Py
+.. _roadmap: https://github.com/ReactionMechanismGenerator/T3/wiki/Roadmap
+.. _semantic: https://semver.org/
+.. _SMILES: https://en.wikipedia.org/wiki/Simplified_molecular-input_line-entry_system
+.. _truhlar: https://dx.doi.org/10.1016/j.cpc.2016.09.004
```
Comment (alongd) [inline docs/source/links.txt:17]: Are all these links being used in out docs? let's leave only the ones that are

File: docs/source/installation.rst [hunk 7]
```diff
@@ -0,0 +1,110 @@
+.. _installation:
+
+Installation instructions
+=========================
+
+Note: T3 was only tested on Linux (Ubuntu_ 18.04.1 LTS) and Mac machines. We don't expect it to work on Windows.
+T3 can be installed on a server, as well as on your local desktop / laptop, submitting jobs to the server/s.
```
Comment (alongd) [inline docs/source/installation.rst:7]: T3 doesn't require RMG to be installed, it's a conda dependency

File: docs/source/installation.rst [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+Note that T3 requires installation of both `RMG <http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/index.html>`_, Reaction Mechanism Generator, and `ARC <https://reactionmechanismgenerator.github.io/ARC/installation.html>`_, Automated Rate calculator.
```
Comment (alongd) [inline docs/source/installation.rst:8]: note that ARC should be installed on the same machine T3 is installed on

File: docs/source/installation.rst [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+
+    export PYTHONPATH=$PYTHONPATH:~/Path/to/T3/
+
+__ anaconda_
+
+
+Install dependencies
+^^^^^^^^^^^^^^^^^^^^
+
+- Install the latest DEVELOPER version of RMG (which has Arkane).
```
Comment (alongd) [inline docs/source/installation.rst:32]: no need to install RMG for T3 (though the user may do so - perhaps we should have two env files, with and w/o `rmgpy`)

File: docs/source/installation.rst [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+     source activate t3_env
+
+
+Add T3 aliases to your .bashrc (for convenience)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Some optional, but convenient, aliases are listed below (make sure to change `~/Path/to/T3/` accordingly).
+Add these to your ``.bashrc`` file, which can be edited by typing, e.g., ``nanoe ~./bashrc``)::
+	
+	export t3_path=$HOME`/Path/to/T3'
+	alias t3_env='source activate t3_env'
```
Comment (alongd) [inline docs/source/installation.rst:58]: let's recommend `t3e` instead

File: docs/source/installation.rst [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+    conda env create -f environment.yml
+
+  Activate the T3 environment every time before you run T3::
+
+     source activate t3_env
+
+
+Add T3 aliases to your .bashrc (for convenience)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Some optional, but convenient, aliases are listed below (make sure to change `~/Path/to/T3/` accordingly).
```
Comment (alongd) [inline docs/source/installation.rst:54]: here we have `~/`, below we don't
although it's a dummy, let's be consistent

File: docs/source/installation.rst (hunk 7 above)
Comment (kspieks) [reply docs/source/installation.rst:7]: My bad. I didn’t pay attention to the environment file

File: docs/source/installation.rst (hunk 11 above)
Comment (kspieks) [reply docs/source/installation.rst:54]: I made them all include `~/` now. Good catch thanks!

File: docs/source/installation.rst (hunk 9 above)
Comment (kspieks) [reply docs/source/installation.rst:32]: I think including RMG in the environment file made sense. I just didn't pay attention before...

**Comment by kspieks** [comment]
Thanks for reviewing this PR! I incorporated your feedback by squashing it into my previous commits. Please let me know if I should edit anything else :)

File: docs/source/index.rst [hunk 12]
```diff
@@ -0,0 +1,40 @@
+T3 - The RMG-ARC Tandem Tool v\ |release|
+============================================
+
+.. image:: t3.png
+   :align: center
+
+**T3 - RMG-ARC Tandem Tool** is a software for automating
```
Comment (alongd) [inline docs/source/index.rst:7]: should be:
`**T3 - The RMG-ARC Tandem Tool** is a software for automating`

File: docs/source/release.rst [hunk 13]
```diff
@@ -0,0 +1,23 @@
+.. _release:
+
+Release notes
+=============
+
+T3 0.1.0
+^^^^^^^^^
```
Comment (alongd) [inline docs/source/release.rst:7]: these `^^^` markings must be the same length as the text above them

File: docs/source/release.rst [hunk 14]
```diff
@@ -0,0 +1,23 @@
...
+
+This is the first version of T3.
+
+T3 is an RMG-ARC Tandem Tool for automated kinetic model generation and refinement.
```
Comment (alongd) [inline docs/source/release.rst:11]: I think:
"T3 is a tool for automated kinetic model generation and refinement."

File: tandem/utils.py [hunk 15]
```diff
@@ -5,6 +5,7 @@
 import os
 from typing import Optional, Union
 
+VERSION='0.1.0'
```
Comment (alongd) [inline tandem/utils.py:8]: please add spaces before and after the `=` operator

File: docs/source/cite.rst [hunk 16]
```diff
@@ -0,0 +1,10 @@
+.. _cite:
+
+How to cite T3
+===============
+
+Text form::
+
+    A. Grinberg Dana, H. Wu, X. Dong, K.A. Spiekermann, W.H. Green, The RMG-ARC Tandem Tool (T3), version 0.1.0, https://github.com/ReactionMechanismGenerator/T3
```
Comment (alongd) [inline docs/source/cite.rst:8]: I suggest:
" A. Grinberg Dana, K.A. Spiekermann, X. Dong, H. Wu, W.H. Green, The RMG-ARC Tandem Tool (T3) for automated kinetic model generation and refinement, version 0.1.0, source code: https://github.com/ReactionMechanismGenerator/T3"

File: docs/source/credits.rst [hunk 17]
```diff
@@ -0,0 +1,19 @@
+.. _credits:
+
+Credits
+=======
+
+T3 is developed at the `Green Group`__ at MIT.
+
+__ greengp_
+
+T3's contributors are:
```
Comment (alongd) [inline docs/source/credits.rst:10]: please change the order to align with the citation

File: docs/source/installation.rst [hunk 18]
```diff
@@ -0,0 +1,106 @@
...
+
+    conda env create -f environment.yml
+
+  Activate the T3 environment every time before you run T3::
+
+     source activate t3_env
+
+
+Add T3 aliases to your .bashrc (for convenience)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (alongd) [inline docs/source/installation.rst:49]: please take a look at all the `^^^` line lengths

**Comment by kspieks** [comment]
Thanks for reviewing and for your close attention to detail (sorry to miss some things). I'll update this tomorrow

**Comment by kspieks** [comment]
I incorporated your feedback by squashing it into my previous commits. Let me know if I can add/edit anything else :)

File: docs/source/installation.rst [hunk 19]
```diff
@@ -0,0 +1,106 @@
...
+
+
+.. _path:
+
+Clone and setup path
+^^^^^^^^^^^^^^^^^^^^
+
+- Download and install the `Anaconda Python Platform`__ for Python 3.7 or higher if you haven't already.
+- Get git if you don't have it already by typing sudo apt-get install git in a terminal.
+- Clone T3's repository to by typing the following command in the desired folder (e.g., under `~/home/Code/`)::
```
Comment (alongd) [inline docs/source/installation.rst:18]: remove extra `to`

File: docs/source/installation.rst [hunk 20]
```diff
@@ -0,0 +1,106 @@
...
+
+  Activate the T3 environment every time before you run T3::
+
+     source activate t3_env
+
+
+Add T3 aliases to your .bashrc (for convenience)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Some optional, but convenient, aliases are listed below (make sure to change `~/Path/to/T3/` accordingly).
+Add these to your ``.bashrc`` file, which can be edited by typing, e.g., ``nanoe ~./bashrc``)::
```
Comment (alongd) [inline docs/source/installation.rst:51]: I think it should be `nano`, not `nanoe`

File: docs/source/cite.rst [hunk 21]
```diff
@@ -0,0 +1,8 @@
+.. _cite:
+
+How to cite T3
+==============
+
+Text form::
+
+    A. Grinberg Dana, K.A. Spiekermann, X. Dong, H. Wu, W.H. Green, The RMG-ARC Tandem Tool (T3) for automated kinetic model generation and refinement, version 0.1.0, source code: https://github.com/ReactionMechanismGenerator/T3
```
Comment (alongd) [inline docs/source/cite.rst:8]: Can we add some reasonable line breaks for better readability?

**Comment by alongd** [comment]
I got the following warning when compiling locally: `source/links.txt:2: WARNING: Duplicate explicit target name: "arc".` Could you take a look?

Also, do you think we should add the API docs in this PR, or in a separate one? (should be relatively easy, I think)

File: docs/source/installation.rst (hunk 19 above)
Comment (kspieks) [reply docs/source/installation.rst:18]: My bad. Thanks for catching that

File: docs/source/installation.rst (hunk 20 above)
Comment (kspieks) [reply docs/source/installation.rst:51]: Indeed, another sloppy typo on my end..

File: docs/source/cite.rst (hunk 21 above)
Comment (kspieks) [reply docs/source/cite.rst:8]: Added some line breaks!

**Comment by kspieks** [comment]
Fixed the warning! I had mistakenly defined a link for ARC in installation.rst as well as in links.txt which had caused the warning.

I added the API to this PR and also added output and running rst files. Thanks so much for reviewing!

**Comment by kspieks** [comment]
Of course! Τhe comments were helpful, and I appreciate your feedback!

**Pull request merged**